### PR TITLE
ci: add stackblitz templates

### PR DESCRIPTION
**Pull request by Wxh16144**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

ref: https://github.com/ant-design/ant-design/pull/49647#issuecomment-2197884871

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      --     |
| 🇨🇳 Chinese |    添加  stackblitz 默认模版      |

### CODE REVIEW

File: .github/workflows/pkg.pr.new.yml [hunk 1]
```diff
@@ -1,8 +1,14 @@
-name: Publish Any Commit
-on: [push, pull_request]
+name: Continuous Releases
+
+on:
+  workflow_run:
+    workflows: ["✅ test"]
+    types:
+      - completed
```
Comment (afc163) [inline .github/workflows/pkg.pr.new.yml:7]: 为啥要加这个

Comment (Wxh16144) [reply .github/workflows/pkg.pr.new.yml:7]: 先前的触发条件是每次 push 都会触发 ci，加上我们后面的 examples 也会越来越多，每次 commit 未免有些浪费资源了， 不能逮着 stackblitz 使劲薅。

所以改成，每次 test 流水线成功后，再去上传资源，并且可以利用 test 跑的一些 node_modules cache，不用每次都 npm install

Comment (afc163) [reply .github/workflows/pkg.pr.new.yml:7]: 我希望测试没跑过也发个版本，这样方便提前验证，而不是什么都搞好了才发。

Comment (Wxh16144) [reply .github/workflows/pkg.pr.new.yml:7]: ok. 我 revert 一下

**Review by afc163 — APPROVED** [review]

**Pull request merged**